scripts/decrypt-el-space-release.sh

```diff
@@ -3,3 +3,4 @@ openssl aes-256-cbc -d -k "$FABRIC_GOOGLE_PASSWORD" -in el-space-app/src/release
 openssl aes-256-cbc -d -k "$FABRIC_GOOGLE_PASSWORD" -in el-space-app/src/release/google-signing-credentials.gradle.enc -out el-space-app/src/release/google-signing-credentials.gradle
 openssl aes-256-cbc -d -k "$FABRIC_GOOGLE_PASSWORD" -in el-space-app/src/release/google-play-api-key.json.enc -out el-space-app/src/release/google-play-api-key.json
 openssl aes-256-cbc -d -k "$FABRIC_GOOGLE_PASSWORD" -in el-space-app/src/release/release-keystore.jks.enc -out el-space-app/src/release/release-keystore.jks
+openssl aes-256-cbc -d -k "$PUSHER_PSW" -in el-space-android/pusher.properties.enc -out el-space-android/pusher.properties
```

Comment:
You trigger decryption twice. Once in decrypt-el-debate-release.sh and the second time here. Create separate script and move decryption there.